target/linux/generic/config-5.15

```diff
@@ -6868,7 +6878,9 @@ CONFIG_USB_GADGET_VBUS_DRAW=2
 # CONFIG_USB_MSI2500 is not set
 # CONFIG_USB_MSM_OTG is not set
 # CONFIG_USB_MTU3 is not set
+# CONFIG_USB_MUSB_GADGET is not set
```

Comment:
Please also remove these configurations for the target kernel configurations:
```
target/linux/mediatek/mt7623/config-5.10:# CONFIG_USB_MUSB_GADGET is not set
target/linux/mediatek/mt7623/config-5.15:# CONFIG_USB_MUSB_GADGET is not set
target/linux/omap/config-5.10:# CONFIG_USB_MUSB_GADGET is not set
target/linux/sunxi/cortexa53/config-5.10:# CONFIG_USB_MUSB_GADGET is not set
target/linux/sunxi/cortexa53/config-5.15:# CONFIG_USB_MUSB_GADGET is not set
target/linux/sunxi/cortexa7/config-5.10:# CONFIG_USB_MUSB_GADGET is not set
target/linux/sunxi/cortexa7/config-5.15:# CONFIG_USB_MUSB_GADGET is not set
```

I did not check the other configurations. Please remove the configurations from the individual target configurations when adding them to generic.

Reply:
donw